android/versioned-abis/expoview-abi39_0_0/src/main/java/abi39_0_0/expo/modules/updates/UpdatesModule.java

```diff
@@ -137,14 +137,14 @@ public void onSuccess(Manifest manifest) {
             // this shouldn't ever happen, but if we don't have anything to compare
             // the new manifest to, let the user know an update is available
             updateInfo.putBoolean("isAvailable", true);
-            updateInfo.putString("manifestString", manifest.getRawManifestJson().toString());
+            updateInfo.putString("manifestString", manifest.getRawManifestJson().getRawJson().toString());
```

Comment:
It looks like maybe the iOS versioned module classes need to have the analogous fix applied too?

Reply:
https://github.com/expo/expo/pull/12631 didn't modify any versioned module classes, though now I'm not sure I am completely clear on what you meant by not needing to modify the versioned module classes if the interface remains the same for `EXUpdatesModule` (from our discussion on slack).

If we do need to version this code I can do that in a follow-up since it'll be a multiple-week-long effort unless I can get the single file version of `et add-sdk` working, but even then it'll take a long time.

On android I made the changes to all versioned code since it was a far simpler because a lot of the updates stuff isn't versioned on android.